core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -26,6 +26,7 @@ case class ApplicationSummaryInfo(
     rapidsProps: Seq[RapidsPropertyProfileResult],
     rapidsJar: Seq[RapidsJarProfileResult],
     sqlMetrics: Seq[SQLAccumProfileResults],
+    gpuMetrics: Seq[AccumProfileResults],
```

Comment:
not `gpuMetrics` anymore.
Also, this is very large array. I am not sure if this is necessary to store this as a field in memory and keep it around.
If `generateStageLevelAccums` is defined in `AppSQLPlanAnalyzer`. Then, it could be called to generate the report without storing it as an extra field in ProfileResult.
Unless, we need this somewhere else.

Reply:
Thanks @amahussein ! renamed gpuMetrics to stageMetrics. As discussed with you offline, the code path for generating Qualification output files and Profile output files are different where the Profile outputwriter doesn't take ApplicationInfo as an argument. So keeping it as it is for now.